### PR TITLE
[astronomer/issues#545] Add taints for dynamic-pods NodePool

### PR DESCRIPTION
**Pull request by kaxil**
Issue: https://github.com/astronomer/issues/issues/545

### CODE REVIEW

**Comment by kaxil** [comment]
What do you guys think about this one @sjmiller609 @schnie ?

**Comment by schnie** [comment]
Seems ok to me. Looks like it may need an update. I'll let @sjmiller609 merge it in.

**Comment by kaxil** [comment]
> Seems ok to me. Looks like it may need an update. I'll let @sjmiller609 merge it in.

Rebased it against master

File: locals.tf [hunk 1]
```diff
@@ -159,6 +159,16 @@ astronomer:
...
             value: gvisor
 %{endif}
+%{if var.create_dynamic_pods_nodepool == true}
+          podMutation:
+            tolerations:
+              - key: "dynamic-pods"
+                operator: "Equal"
+                value: "true"
+                effect: "NoSchedule"
+            nodeSelectors:
```
Comment (kaxil) [inline locals.tf]: I have updated these configs @schnie based on https://github.com/astronomer/helm.astronomer.io/pull/237/files

Comment (kaxil) [reply locals.tf]: Added in `nodeSelectors` !

**Comment by schnie** [comment]
LGTM. @sjmiller609 you good with it?

**Pull request merged**